pkg/webhook/pod/pods.go

```diff
@@ -91,7 +96,9 @@ func (pc *PodAdmissionControl) AdmitPods(ar v1beta1.AdmissionReview) *v1beta1.Ad
 //// otherwise we will check it decided by component type.
 func (pc *PodAdmissionControl) admitDeletePods(name, namespace string) *v1beta1.AdmissionResponse {
 
-	pod, err := pc.podLister.Pods(namespace).Get(name)
+	// We would update pod annotations if they were deleted member by admission controller,
+	// so we shall find this pod from apiServer considering getting latest pod info.
+	pod, err := pc.kubeCli.CoreV1().Pods(namespace).Get(name, metav1.GetOptions{})
```

Comment:
We would update pod annotations if they were deleted member by admission controller, so we shall find this pod from apiServer considering getting latest pod info.

Reply:
What if it's changed after this get?

Reply:
In this case, we only need to get pod.name and pod.annotations ( defer deleting specificlly) from pod info. If `delfer deleting` is added, it wouldn't be remove again.